src/ruby/Cone.i

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
I think you don't need to add this file. @azeey ?

Reply:
Does it hurt to have?

Reply:
I guess if we need to get rid of it re-open?